### PR TITLE
Page builder adjustments

### PR DESCRIPTION
**Pull request by aachavarria**
adding snackbar to PreviewConcierge.tsx;
adding desktop upload assets progress mask

### CODE REVIEW

File: ui/app/src/components/CrafterCMSNextBridge.tsx [hunk 1]
```diff
@@ -82,7 +82,7 @@ function CrafterCMSNextBridge(props: PropsWithChildren<{ isLegacy?: boolean }>)
     <Provider store={store}>
       <RawIntlProvider value={intl}>
         <ThemeProvider theme={theme}>
-          <SnackbarProvider maxSnack={5}>
+          <SnackbarProvider maxSnack={5} anchorOrigin={{ vertical: 'top', horizontal: 'right' }}>
```
Comment (rart) [inline ui/app/src/components/CrafterCMSNextBridge.tsx]: Are these just defaults? Consumers may override?

File: ui/app/src/modules/Preview/PreviewConcierge.tsx [hunk 2]
```diff
@@ -410,20 +411,15 @@ function beginGuestDetection(setSnack): Subscription {
...
+    enqueueSnackbar(<FormattedMessage
+      id="guestDetectionMessage"
+      defaultMessage="Communication with guest site was not detected."
+    />, {
+      action: (key) => <Button
+        key="learnMore" color="secondary" size="small" onClick={() => closeSnackbar(key)}
+      >
+        Learn More
+      </Button>
     });
```
Comment (rart) [inline ui/app/src/modules/Preview/PreviewConcierge.tsx]: Format

File: ui/guest/src/util.tsx [hunk 3]
```diff
@@ -460,6 +466,15 @@ export function retrieveProperty(object: object, prop: string): any {
...
 }
 
+export function deleteProperty(object: object, prop: string): object {
+  return Object.keys(object).reduce((next, key) => {
+    if (key !== prop) {
+      next[key] = object[key];
+    }
+    return next;
+  }, {});
+}
```
Comment (rart) [inline ui/guest/src/util.tsx]: This is sort of equivalent to reversePluckProps. 

A deleteProperty should be more like
```typescript
export function deleteProperty<T, P extends keyof T>(object: T, prop: P): Omit<T, P> {
  delete object[prop];
  return object;
}
```

Notice it removes the prop from the source object.

Comment (rart) [reply ui/guest/src/util.tsx]: Add to app's object.ts too.

**Review by rart — CHANGES_REQUESTED** [review]
Is this still on draft stage on purpose — still being worked?

File: ui/app/src/components/CrafterCMSNextBridge.tsx (hunk 1 above)
Comment (aachavarria) [reply ui/app/src/components/CrafterCMSNextBridge.tsx]: yes

File: ui/app/src/components/BulkUpload.tsx [hunk 4]
```diff
@@ -678,7 +671,8 @@ export default function BulkUpload(props: BulkUploadProps) {
       <DialogHeader
         title={formatMessage(translations.title)}
         subtitle={formatMessage(translations.subtitle)}
-        onClose={dropZoneStatus.status === 'uploading' ? onMinimized : () => onClose(dropZoneStatus)}
+        //onClose={dropZoneStatus.status === 'uploading' ? onMinimized : () => onClose(dropZoneStatus)}
```
Comment (rart) [inline ui/app/src/components/BulkUpload.tsx]: x

File: ui/app/src/modules/Preview/PreviewConcierge.tsx [hunk 5]
```diff
@@ -286,18 +287,29 @@ export function PreviewConcierge(props: any) {
...
+                }
+              });
             },
             (error) => {
-              setSnack({ message: error });
+              console.log(error);
+              enqueueSnackbar('Asset Upload failed.');
             },
             () => {
+              console.log('finished');
```
Comment (rart) [inline ui/app/src/modules/Preview/PreviewConcierge.tsx]: x

**Review by rart — APPROVED** [review]

**Pull request merged**